skopt/space.py

```diff
@@ -324,6 +352,13 @@ def __init__(self, dimensions):
 
         self.dimensions = _dimensions
 
+    def __eq__(self, other):
+        return all([a == b for a, b in zip(self.dimensions, other.dimensions)])
+
+    def __repr__(self):
+        return "Space([{}])".format(
```

Comment:
Not sure as well. What if there are a 1000 dimensions? I would say let's increase the limit to first 20-25 and last 20-25?

Would appreciate what others have to think as well. cc: @betatim